<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>728</width>
    <height>449</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>728</width>
    <height>449</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>8</pointsize>
    <weight>75</weight>
    <italic>false</italic>
    <bold>true</bold>
    <underline>false</underline>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/assets/img/icons/Untitled-2.png</normaloff>:/assets/img/icons/Untitled-2.png</iconset>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>3</number>
    </property>
    <property name="leftMargin">
     <number>3</number>
    </property>
    <property name="topMargin">
     <number>3</number>
    </property>
    <property name="rightMargin">
     <number>3</number>
    </property>
    <property name="bottomMargin">
     <number>3</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="10,90">
      <property name="spacing">
       <number>3</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="Header_layout" stretch="0">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QWidget" name="horizontalWidget" native="true">
          <property name="styleSheet">
           <string notr="true">#horizontalWidget{
	background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:1.107, fx:0.5, fy:0.5, stop:0.431818 rgba(89, 65, 169, 255), stop:0.772727 rgba(255, 255, 255, 255));
	border-radius:10px;
background-color: rgb(89, 65, 169);/*
background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:1.107, fx:0.5, fy:0.5, stop:0.259887 rgba(89, 65, 169, 255), stop:1 rgba(255, 255, 255, 255));*/
color: rgb(229, 212, 237)
}
</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout" stretch="10,80,10">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="header">
             <property name="font">
              <font>
               <pointsize>15</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(255, 255, 255);</string>
             </property>
             <property name="text">
              <string>BookIT</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="margin">
              <number>0</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(255, 255, 255);</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="res.qrc">
               <normaloff>:/assets/img/icons/information-button.png</normaloff>:/assets/img/icons/information-button.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>26</width>
               <height>26</height>
              </size>
             </property>
             <property name="shortcut">
              <string>Ctrl+I</string>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="fullbody" stretch="0">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QWidget" name="widget" native="true">
          <layout class="QHBoxLayout" name="fullbodyWid" stretch="0,0,0">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QWidget" name="menu_container" native="true">
             <property name="styleSheet">
              <string notr="true">#menu_container{
	border-radius:10px;
	background-color: rgb(89, 65, 169);
	color: rgb(229, 212, 237);
}
QWidget{
color: rgb(229, 212, 237);

}
</string>
             </property>
             <layout class="QVBoxLayout" name="menu_layout_2" stretch="10,10,10,10,10">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>6</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="Movies">
                <property name="font">
                 <font>
                  <pointsize>17</pointsize>
                  <weight>75</weight>
                  <bold>true</bold>
                  <underline>false</underline>
                 </font>
                </property>
                <property name="frameShape">
                 <enum>QFrame::NoFrame</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Plain</enum>
                </property>
                <property name="text">
                 <string>Movies</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="Events">
                <property name="font">
                 <font>
                  <pointsize>17</pointsize>
                  <weight>50</weight>
                  <bold>false</bold>
                  <underline>false</underline>
                 </font>
                </property>
                <property name="frameShape">
                 <enum>QFrame::NoFrame</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Plain</enum>
                </property>
                <property name="text">
                 <string>Events</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="Sports">
                <property name="font">
                 <font>
                  <pointsize>17</pointsize>
                  <weight>50</weight>
                  <bold>false</bold>
                  <underline>false</underline>
                 </font>
                </property>
                <property name="frameShape">
                 <enum>QFrame::NoFrame</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Plain</enum>
                </property>
                <property name="text">
                 <string>Sports</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="Plays">
                <property name="font">
                 <font>
                  <pointsize>17</pointsize>
                  <weight>50</weight>
                  <bold>false</bold>
                  <underline>false</underline>
                 </font>
                </property>
                <property name="frameShape">
                 <enum>QFrame::NoFrame</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Plain</enum>
                </property>
                <property name="text">
                 <string>Plays</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="watchList">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="font">
                 <font>
                  <pointsize>17</pointsize>
                  <weight>50</weight>
                  <bold>false</bold>
                  <underline>false</underline>
                 </font>
                </property>
                <property name="frameShape">
                 <enum>QFrame::NoFrame</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Plain</enum>
                </property>
                <property name="text">
                 <string>WatchList</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
                <property name="wordWrap">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <property name="spacing">
              <number>0</number>
             </property>
             <item>
              <widget class="QPushButton" name="menubtn">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>13</width>
                 <height>0</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>6</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">border-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));</string>
               </property>
               <property name="text">
                <string>&lt;</string>
               </property>
               <property name="iconSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="flat">
                <bool>false</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="showCaseBody_2" stretch="10,90">
             <item>
              <layout class="QHBoxLayout" name="filter_layout_2">
               <property name="leftMargin">
                <number>3</number>
               </property>
               <property name="topMargin">
                <number>3</number>
               </property>
               <property name="rightMargin">
                <number>3</number>
               </property>
               <property name="bottomMargin">
                <number>3</number>
               </property>
               <item>
                <widget class="QComboBox" name="cats">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">	border-radius:10px;
border:3px solid;
border-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(89, 65, 169,255));
padding:5px;</string>
                 </property>
                 <property name="frame">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="show_layout54646_2">
               <item>
                <widget class="QScrollArea" name="scrollArea">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true"/>
                 </property>
                 <property name="frameShape">
                  <enum>QFrame::NoFrame</enum>
                 </property>
                 <property name="widgetResizable">
                  <bool>true</bool>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                 <widget class="QWidget" name="scrollAreaWidgetContents_2">
                  <property name="geometry">
                   <rect>
                    <x>0</x>
                    <y>0</y>
                    <width>594</width>
                    <height>344</height>
                   </rect>
                  </property>
                  <layout class="QVBoxLayout" name="show_layout">
                   <property name="spacing">
                    <number>2</number>
                   </property>
                   <property name="leftMargin">
                    <number>2</number>
                   </property>
                   <property name="topMargin">
                    <number>2</number>
                   </property>
                   <property name="rightMargin">
                    <number>2</number>
                   </property>
                   <property name="bottomMargin">
                    <number>2</number>
                   </property>
                  </layout>
                 </widget>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
